lib/ansible/module_utils/basic.py

```diff
@@ -1993,7 +1993,7 @@ def atomic_move(self, src, dest, unsafe_writes=False):
                             try:
                                 try:
                                     out_dest = open(b_dest, 'wb')
-                                    in_src = open(b_src, 'rb')
+                                    in_src = open(b_tmp_dest_name, 'rb')
```

Comment:
are you sure this is what we want? b_src should be what we 'tried to copy to b_tmp_dest_name'